cypress/PageObjects/HomePage.js

```diff
@@ -4,6 +4,10 @@ class HomePage {
     return cy.get('[dir="auto"]').eq(0)
   }
 
+  get supportButton() {
+    return cy.get('[data-focusable="true"]:nth-child(7)')
```

Comment:
we need a better selector

Reply:
well?

Reply:
it fixed in another PR https://github.com/GoodDollar/GoodDAPP/pull/1196

Reply:
so we dont need this PR?